tools/negative_motif_finder/NMF.xml

```diff
@@ -0,0 +1,49 @@
+<tool id="kinamine_java" name="Negative-Motif-Finder 7_to_7" version="1.3.0">
+    <description>discover unphosphorylated substrates</description>
+    <requirements>
+       <requirement type="package">R</requirement>
+    </requirements>
+    <command><![CDATA[
+		ln -s '$FASTA' input1.tabular && 
+        ln -s '$positives' input2.csv &&
+		ln -s '$SBF' input3.csv &&
+        Rscript '$__tool_directory__/NMF-working-1-9-18.R'
+    ]]></command>
+    <inputs>
+		<param format="csv" name="SBF" type="data" label="Substrate Background Frequency"/>
+        <param format="tabular" name="FASTA" type="data" label="Human Proteome Reference"/>
+        <param format="csv" name="positives" type="data" label="Positive/Phosphorylated Substrate List"/>
+		<param name="outGroup" type="text" value="kinase" label="Kinase Name"/>
+    </inputs>      
+    <outputs>
+        <data format="csv" name="Negatives" from_work_dir="output1.csv" label="${outGroup}_Negative Motifs.csv"/>
+    </outputs>
+    <tests>
+        <test>
+            <param name="FASTA" ftype="csv" value="reference.csv"/>
+            <param name="positives" ftype="csv" value="substrates.csv"/>
+            <param name="SBF" ftype="csv" value="SBF.csv"/>
+			<output name="Negatives" file="negatives.csv"/>

```

Comment:
please correct your indentation ... we prefer 4 spaces instead of a TAB. This line here for example looks odd.